### ISSUE TITLE
Add specifications with model in levels

### ISSUE THREAD

**Issue by diegogentilepassaro** [diegogentilepassaro/min_wage_rent#60]
[Goal]
Have a working directory with different versions of our baseline model but in levels instead of in first differences. 

[Notes]
- Make sure that we try models using within zipcode variation and model using within state and across zipcodes variation.

**Comment by diegogentilepassaro** [comment]
@santiagohermo as per our conversation we want to estimate a time-demeaned model in levels. My interpretation is that a model like that is equivalent to a fixed effect model in levels. Assumptions for these kinds of models and the first differenced model that we were doing are different though. If the error terms are homoskedastic with no serial correlation, the fixed effects estimator is more efficient. If they are persistent, for example they follow a random walk, the first difference estimator is more efficient.
Do you agree?

**Comment by diegogentilepassaro** [comment]
@santiagohermo and @gabrieleborg I have built a first pass of a directory that outputs level models with different sets of fixed effects. The compiled table lives in the branch `issue_60` under staged_output.pdf and it looks like:

![Screen Shot 2020-11-02 at 5 46 38 PM](https://user-images.githubusercontent.com/20582494/97917533-5e720380-1d33-11eb-9caa-dbba3bd7758b.png)

All model absorb time fixed effects. As for the models with zipcode fixed effects results look a little different to our baseline but they are not too bad. In all models, regardless of the set of FEs, results are really sensitive to the specification. I interpret this as evidence that local shocks in rents are pretty persistent and therefore the level model is really sensitive to how we allow for dynamics (if allowing for it).

Another note: with state FEs either effects are much bigger (columns 7 and 9) or  there is a huge pretrend. I am not sure how to interpret this yet, but my first hunch is that within a state zipcodes are really heterogeneous so comparisons over time across them are less clean.

**Comment by santiagohermo** [comment]
Thanks @diegogentilepassaro! 

- As for your [first comment](https://github.com/diegogentilepassaro/min_wage_rent/issues/60#issuecomment-720638858). This is how I view it. We have a two-way FE model. We can either estimate (i) in first-differences, (ii) by individual demeaning, (iii) in levels. I agree that (ii) and (iii) may be equivalent in the sense that they yield the same coefficients (individual demeaning may be what STATA is doing in the background?). But are they equivalent in terms of the std errors? What about the assumptions needed for identification? Something to think about. In any case, I think adding one column with demeaning within zipcodes wouldn't hurt. At least we can confirm our intuition!

- Regarding [the results](https://github.com/diegogentilepassaro/min_wage_rent/issues/60#issuecomment-720718208). 
  - I think the nicest results are (1) and (2), which sort of confirm the FD. The county level results are also nice. I don't really understand why restricting the pretrend makes the same period coefficient non-significant.
   - Your hyphotesis about the behavior of the shocks is interesting. Maybe a way of testing it would be to add lagged rents to the models?
  -  No idea why the state look so weird. Maybe at that level our econometric assumptions don't hold.

A lot of things to think about @diegogentilepassaro! Looking forward to our meeting :)

**Comment by santiagohermo** [comment]
See Wooldrige section 10.6.3 for a test of serial correlation in the context of fixed effects model @diegogentilepassaro

**Comment by diegogentilepassaro** [comment]
@gabrieleborg and @santiagohermo, I finally got to finish this task! In the end, I produced two tables both live in the staged_output part of `issue__60` branch. 

The first, and most important one compares the TWFE model in levels and it's analogous in first differences. According to Wooldridge, these two models rely on different assumptions. The TWFE model needs assumption FE.3: errors are not autocorrelated. Instead, the FD model can allow for autocorrelation in the errors (we are always talking about the level model errors). One can test for AR 1 autocorrelation in the level errors by estimating the FD model, saving the residuals, and regressing those residuals on their lag. Call that coefficient `\rho` Under assumption FE.3. being true it should be the case that `\rho = -0.5`. Testing that hypothesis is our autocorrelation test. In our data, we very comfortably reject the null, and therefore we now have a formal justification for the FD model. Of course this model still needs some assumptions, but those are already discussed in the paper! I report the table below. Note that I added notes in case we want to send this to an appendix. 
![Screen Shot 2020-12-02 at 7 45 29 PM](https://user-images.githubusercontent.com/20582494/100940567-f1d75980-34d6-11eb-9ade-0bea5a05c9d9.png)

The second table is just the level model with different sets of geography FEs. The punchline is that when comparing within zipcodes or across zipcodes within a county we get similar results, is just that the former is more precise. When we compare across zipcodes within the state we get tremendously big results. My hypothesis is that is driven by huge zipcode heterogeneity within the state. In column (4), I add zipcode-specific linear trends to the model with state FE and the effect vanishes. I don't think that this table will make it to the draft, but it is good to have it here as a reference.
![Screen Shot 2020-12-02 at 7 47 58 PM](https://user-images.githubusercontent.com/20582494/100940741-4975c500-34d7-11eb-9942-eb8b3b817252.png)

I will send this issue to PR!
